eclass/cvs.eclass

```diff
@@ -192,7 +198,9 @@ if [[ ${ECVS_AUTH} == "ext" ]] ; then
 	if [[ ${CVS_RSH} != "ssh" ]] ; then
 		die "Support for ext auth with clients other than ssh has not been implemented yet"
 	fi
-	BDEPEND+=" net-misc/openssh"
+	PYTHON_COMPAT=( python3_{10..13} )
```

Comment:
Unpopular opinion but sometimes we really should just write code that is basic enough that we can reasonably assume it works regardless of python feature release, and not require a specific COMPAT.

The presence of hammers makes everything look like a nail. Just because nails are an extremely common building material doesn't mean it's the only type.

Reply:
Replace it by `BDEPEND="dev-lang/python"` then? We need something there because Python is not in the system set.

Reply:
> Replace it by `BDEPEND="dev-lang/python"` then? We need something there because Python is not in the system set.

Well, CI doesn't seem to like it:
```
9999	MissingPythonEclass	missing python-any-r1 eclass usage for BDEPEND="dev-lang/python"v-lang/python"
```